tests/jupyterhub_api/SessionUtils.py

```diff
@@ -0,0 +1,65 @@
+import os
+import subprocess
+import requests
+import json
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+class CreateSession:
+
+    def __init__(self, port, users, path):
+        self.port = port
+        self.users = users
+        self.path = path
+        self.main_url = "https://localhost:" + str(self.port) + "/hub/api/"
+        self.exit = 0
+        self.token =0
+
+
+
+    def create_token(self):
+        owd = os.getpwd()
+        os.chdir(os.path.expanduser(self.path))
```

Comment:
I have changed the directory here to run the command "jupyterhub token dummy_admin".  If the command is executed elsewhere token will not be generated. 
Example:
```
[root@204fc389c0f6 /]# jupyterhub token dummy_admin
[W 2019-06-30 08:30:04.727 JupyterHub app:1173] No admin users, admin interface will be unavailable.
[W 2019-06-30 08:30:04.728 JupyterHub app:1174] Add any administrative users to `c.Authenticator.admin_users` in config.
[I 2019-06-30 08:30:04.728 JupyterHub app:1201] Not using whitelist. Any authenticated user will be allowed.
No such user: dummy_admin
```
```
[root@204fc389c0f6 /]# cd  srv/jupyterhub
[root@204fc389c0f6 jupyterhub]# jupyterhub token dummy_admin
Authenticator: Using LDAP
[D 2019-06-30 10:34:12.647 JupyterHub app:1071] Connecting to db: sqlite:////srv/jupyterhub/jupyterhub.sqlite
[I 2019-06-30 10:34:12.704 JupyterHub app:1201] Not using whitelist. Any authenticated user will be allowed.
e509b6026dcd4a77b8a41b68c7c0be56
```